src/docs/sdk/performance/dynamic-sampling-context.mdx

```diff
@@ -0,0 +1,124 @@
+---
+title: "Dynamic Sampling Context (Experimental)"
+---
+
+<Alert level="warning">
+
+This page is under active development.
+Specifications are not final and subject to change.
+Anything that sounds fishy probably is.
+
+</Alert>
+
+Until now, traces sampling was only done through a `sample_rate` option in the SDKs.
+This has quite a few drawbacks for users of Sentry SDKs:
```

Comment:
Isn't `tracesSampleRate`? if it's for traces.
`sampleRate` is only for events.

Reply:
Yes sir you are correct. Thanks for pointing this out. Fixed in 6cdc5699e8486b30c823cefc58fd215dfe38b1a7.

Reply:
Nice, I'd look up the usage of this name in the docs tho, there are more references to `sample_rate` or `sampleRate`, such as `sentry-samplerate`.

Reply:
Good point. I changed it to camel case and added links to the relevant sections in the docs (where it is also in camel case): 1a27b856d548737ea824e9f09404c3d44ef9c933